pints/plot/_histogram.py

```diff
@@ -88,33 +88,38 @@ def histogram(
         squeeze=False,    # Tell matlab to always return a 2d axes object
     )
 
+    # Find ranges across all samples
```

Comment:
Should this bit go in a private(?) method so that both methods can share it?

Reply:
I mean, it _could_ but I'm not sure it'll be used anywhere else, so think can just leave it as is.